samples/AMT-DataMart.sql

```diff
@@ -108,6 +108,12 @@ SELECT *
 INTO [$(DataMartDB)].[analytics].[stg_asmt_StudentAssessmentFact]
 FROM [$(OdsDb)].[analytics].[asmt_StudentAssessmentFact]
 
+-- CHRAB Collection
+
+SELECT *
+INTO [$(DataMartDB)].[analytics].[stg_chrab_ChronicAbsenteeismAttendanceFact]
+FROM [$(OdsDb)].[analytics].[chrab_ChronicAbsenteeismAttendanceFact]
```

Comment:
This is the view with terrible performance, right? Is it really necessary to include it in the scope of this ticket? Since the views are being materialized into the datamart database, if this fact view is actually needed, then it might be better to build a slightly different process for it: instead of querying the view itself, work on a two-stage solution where you materialize _some_ of the data into an intermediate table, and then a second query to build the actual fact view.

Reply:
New ticket created to work on chrab_ChronicAbsenteeismAttendanceFact view [BIA-940](https://tracker.ed-fi.org/browse/BIA-940)